extensions/clusters/import.go

```diff
@@ -62,8 +62,15 @@ func ImportCluster(client *rancher.Client, cluster *apisV1.Cluster, rest *rest.C
 	ts := client.Session.NewSession()
 	defer ts.Cleanup()
 
+	backoff := kwait.Backoff{
+		Duration: 1 * time.Second,
+		Factor:   1.1,
+		Jitter:   0.1,
+		Steps:    20,
+	}
+
 	var token management.ClusterRegistrationToken
-	err := kwait.Poll(500*time.Millisecond, 2*time.Minute, func() (done bool, err error) {
+	err := kwait.ExponentialBackoff(backoff, func() (finished bool, err error) {
```

Comment:
any specific reason you changed it to a backoff rather than a poll? Just curious for future reference.